net/relayd/src/opnsense/mvc/app/models/OPNsense/Relayd/Menu/Menu.xml

```diff
@@ -1,19 +1,8 @@
 <menu>
-    <Services>
-        <LoadBalancer VisibleName="Load Balancer" cssClass="fa fa-truck fa-fw">
-            <Monitors order="10" url="/load_balancer_monitor.php">
-                <Edit url="/load_balancer_monitor_edit.php*" visibility="hidden"/>
-            </Monitors>
-            <PoolSetup order="20" VisibleName="Pool Setup" url="/load_balancer_pool.php">
-                <Edit url="/load_balancer_pool_edit.php*" visibility="hidden"/>
-            </PoolSetup>
-            <VirtualServer order="30" VisibleName="Virtual Server Setup" url="/load_balancer_virtual_server.php">
-                <Edit url="/load_balancer_virtual_server_edit.php*" visibility="hidden"/>
-            </VirtualServer>
-            <Settings order="40" url="/load_balancer_setting.php"/>
-            <PoolStatus order="50" VisibleName="Pool Status" url="/status_lb_pool.php"/>
-            <VirtualServerStatus order="60" VisibleName="Virtual Server Status" url="/status_lb_vs.php"/>
-            <Log VisibleName="Log File" order="100" url="/diag_logs_relayd.php"/>
-        </LoadBalancer>
-    </Services>
+   <Services>
+      <Relayd VisibleName="Load Balancer" cssClass="fa fa-truck fa-fw">
+         <Settings url="/ui/relayd/"/>
+         <Status url="/ui/relayd/status/"/>
```

Comment:
No log file viewer anymore? The file could be kept as is, we do the same for other MVC components until a replacement is found.

Thanks for keeping fa-truck ❤️

Reply:
Huuch!
And the dashboard widget too.
Wasn't aware of that.

Reply:
widget is bonus material, no apparent need unless you think it valuable (haproxy also has no widget)